### PR TITLE
Added automatic testnet command

### LINKED ISSUE #33: Add generate testnet scripts
Add script to `Makefile` to generate testnet automatically. The script will do the follow steps.

1. Init genesis file
2. Update bond denom
3. Create 4 `.desmosd` and `.desmoscli` home directories
4. Create 1 key in each `.desmoscli` home directory
5. Add genesis accounts to all 4 addresses
6. Create gentx with the 4 keys from different directories
7. Copy all gentx to one single gentx directory
8. Collect gentxs
9. Copy the generated genesis file to each `.desmosd/config` directory
10. Update ports to the genesis files in each `.desmosd/config` directory
11. Run `make install`
12. Start 4 nodes with 4 different `.desmosd` home directories

### PR DESCRIPTION
**Pull request by RiccardoM**
## Description
Added the `localnet-start` and `localnet-stop` commands which allow to create a local 4-node testnet using a Docker image. 
Taken and adapted from the [gaia repo](https://github.com/cosmos/gaia/blob/master).

Closes #33 

## Checklist
- [x] Targeted PR against correct branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [x] Re-reviewed `Files changed` in the Github PR explorer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,3 +33,96 @@ Edit `$HOME/.desmosd/config/genesis.json` and update the `bond_denom` from `stak
...
+To update the binary, just rebuild it and restart the nodes:
+
+```
+make build-linux localnet-start
+```
+
+#### Configuration
+
+The `make localnet-start` creates files for a 4-node testnet in `./build` by
+calling the `demosd testnet` command. This outputs a handful of files in the
```
Comment (kwunyeung) [inline README.md]: `desmosd testnet`

File: networks/local/Makefile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Makefile for the "desmosd" docker image.
+
+all:
+	docker build --tag desmos-labs/desmosdnode desmosdnode
```
Comment (kwunyeung) [inline networks/local/Makefile]: The organization I created on Docker Hub is `desmoslabs` (https://hub.docker.com/orgs/desmoslabs). I think the tag has to be `desmoslabs/desmosdnode` here.

Comment (RiccardoM) [reply networks/local/Makefile]: Fixed it!

File: README.md (hunk 1 above)
Comment (RiccardoM) [reply README.md]: Fixed, thanks for the heads up

**Comment by RiccardoM** [comment]
Yeah I'm fine with merging this and opening an issue for the specific error

**Pull request merged**